en/docs/administer/product-level-security-guidelines.md

```diff
@@ -319,4 +319,12 @@ Follow the steps below to change the default credentials.
     
     - Linux/Unix : sh wso2server.sh
     - Windows : wso2server.bat
+
+## Certificate validation for REST APIs
+
+When configuring intermediate certificate validation for REST APIs, it is recommended to add the exempt_contexts parameter and leave it empty. This is because authentication might be broken in your system for the exempted contexts.
```

Comment:
```suggestion
When configuring intermediate certificate validation for REST APIs, it is recommended to add the `exempt_contexts` parameter and leave it empty. This is because authentication might fail for the exempted contexts.
```

Reply:
Addressed with 5b0a50120f0d777484e0fa0eb294fc14fd03f299